app/controllers/api/application_instances_controller.rb

```diff
@@ -3,6 +3,8 @@ class Api::ApplicationInstancesController < Api::ApiApplicationController
   load_and_authorize_resource :application
   load_and_authorize_resource :application_instance, through: :application
 
+  before_action :set_configs, only: [:create, :update]
```

Comment:
Consider `before_action :set_configs, only: %i[create update]` instead as that is the new ruby syntax.